README.md

```diff
@@ -137,7 +137,7 @@ The examples above would be solved like this:
 ```javascript
 class MyClass {
   print() {
-    console.log(@`
+    console.log(String.dedent`
```

Comment:
Merged too soon, I hope @mmkal is ok!

Reply:
:’( I’m not ok! But what can you do. @jridgewell are there notes from the plenary?

Reply:
Notes are released after 1 week. Some of the disapproval was on [Matrix](https://github.com/tc39/how-we-work/blob/master/matrix-guide.md), though, with only 2 expression dissent in the meeting. https://matrixlogs.bakkot.com/TC39_Delegates/2022-03-30#L370-L464 is the matrix logs